packages/progress-bar/ElmStories/ProgressBarStories.elm

```diff
@@ -0,0 +1,27 @@
+module ElmStories.ProgressBarStories exposing (main)
```

Comment:
This file's path doesn't appear to be covered by the `"source-directories"` in the **elm.json** file. Is this a problem? Should this be moved under the **src** directory?

Reply:
Interesting one. It works because `ProgressBarStories` is the `main`, so Elm doesn't need to find it in `source-directories`

Reply:
I think even if the Elm compiler can accept it, this'll break some dev tools like elm-format and the Elm Language Server.